documentation-website/Writerside/topics/Transactions.md

```diff
@@ -316,16 +322,16 @@ transaction {
 
 DSL operations within a transaction create SQL statements, on which commands like *Execute*, *Commit*, and *Rollback*
 are issued. Exposed provides
-the `StatementInterceptor` [interface](https://github.com/JetBrains/Exposed/blob/main/exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/statements/StatementInterceptor.kt)
-that allows you to implement your own logic before and after these specific steps in a statement's lifecycle.
+the [`StatementInterceptor`](https://jetbrains.github.io/Exposed/api/exposed-core/org.jetbrains.exposed.sql.statements/-statement-interceptor/index.html)
+interface that allows you to implement your own logic before and after these specific steps in a statement's lifecycle.
 
 `registerInterceptor()` and `unregisterInterceptor()` can be used to enable and disable a custom interceptor in a single
 transaction.
 
-To use a custom interceptor that acts on all transactions, extend the `GlobalStatementInterceptor` class instead.
-Exposed uses the Java SPI ServiceLoader to discover and load any implementations of this class. In this situation, a new
-file should be created in the *resources* folder
-named:
+To use a custom interceptor that acts on all transactions, extend the
+[`GlobalStatementInterceptor`](https://jetbrains.github.io/Exposed/api/exposed-core/org.jetbrains.exposed.sql.statements/-global-statement-interceptor/index.html)
+class instead. Exposed uses the Java SPI ServiceLoader to discover and load any implementations of this class. In this
```

Comment:
Could you please switch both usages of 'class' with 'interface', for GlobalStatementInterceptor? And 'extend' -> 'implement'.